engine/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -97,6 +86,8 @@ Before following this procedure, you must first meet all the
     }
     ```
 
+    If you need to use the legacy `overlay` driver, specify it instead.
```

Comment:
If you use the `overlay` driver and you specify that kernel check override, what happens?

Reply:
The kernel check override is ONLY for overlay2.